server/server.js

```diff
@@ -0,0 +1,37 @@
+import { fileURLToPath } from 'url';
+import path from 'path';
+import express from 'express';
+import dotenv from 'dotenv';
+import bodyParser from 'body-parser';
+import spicesRoute from './routes/spicesRoute.js'
+// import shoppingListRoute from './routes'
```

Comment:
As a rule, PRs shouldn't have commented out code. It invites questions and distraction from the reviewer which slows the team down.

Please remove all commented out code in this file.